### DISCUSSION TITLE
📣 POLL: What type of resource would you like to see more regarding SP-API?

### DISCUSSION THREAD

**Discussion by rodrifed** [amzn/selling-partner-api-samples#36] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

**Comment by rodrifed** [comment]
#### References:
* Documentation website: https://developer-docs.amazon.com/sp-api/
* Example code (this repository): https://github.com/amzn/selling-partner-api-samples
* Amazon SP-API Developer University YouTube channel: https://www.youtube.com/@amazon-sp-api

**Comment by johnkw** [comment]
Previously the `selling-partner-api-docs` github was a critically valuable resource of open bug reports. Samples are nice I suppose, but I never use samples -- just the docs. Documenting a workflow is more important than a code sample.

But the most valuable thing this github could provide would just be an open "Issues" forum to discuss bugs and get authoritative answers on the status of bugs.

Unfortunately many of the old `selling-partner-api-docs` bug reports were either closed, despite being still open issues, or they were migrated to "Discussions" in the `selling-partner-api-models` repo.  "Discussions" are better than nothing, but a discussion is not a bug report, and it's nice to be able to review the two separately.

**Comment by xh1161797617** [comment]
Sample structures for various types of requests and responses in Feeds

**Reply by leonardonakagawa** [reply]
Hi @xh1161797617 ! 

Thanks for the feedback. Have you already checked the [Listing workflow migration tutorial](https://developer-docs.amazon.com/sp-api/docs/listing-workflow-migration-tutorial) it contains strategies to migrate listings feeds to JSON. 

Is this what you looking for? Or specifically to the Feed creation workflow? 

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support

**Reply by johnkw** [reply]
There currently is no programmatic migration path available whatsoever. See https://github.com/amzn/selling-partner-api-models/issues/4264 and https://github.com/amzn/selling-partner-api-models/issues/4318 for example.

**Reply by xh1161797617** [reply]
It is an example of a request for order cancellation. For example, POST_ORDER_ACKNOWLEDGEMENT_DATA in the Feeds API

**Comment by johnkw** [comment]
As per my prior message, the most valuable thing this github could do, is maintain an active list of known issues/bugs.  Unfortunately it has not done a great job of that over the last year.  For example these 2 blocker bugs have been closed without a resolution as "not planned":

https://github.com/amzn/selling-partner-api-models/issues/4264
https://github.com/amzn/selling-partner-api-models/issues/4318

It would be great to reopen those, and, in general, leave bugs open until they're fixed.
